Library/Homebrew/cmd/install.rb

```diff
@@ -261,7 +261,9 @@ def install_formula(f)
     f.print_tap_action
 
     fi = FormulaInstaller.new(f)
-    fi.options             = f.build.used_options
+    build_options          = f.build
```

Comment:
Pull this above the `fi` definition like in `reinstall.rb` to make it clearer and don't indent it.